projects/transformers/callbacks/metrics.py

```diff
@@ -58,6 +58,8 @@ def __init__(self, n_eval_sets=1, sparsity_tolerance=0.01):
     def on_evaluate(self, args, state, control, metrics, **kwargs):
         """Update eval metrics and possibly step counter, sparsity, and lr"""
 
+        print(f"In Metrics Callback. Here are the metrics: {metrics}")
```

Comment:
Can we change this to the logging, so it can't be more easily controlled by the global verbosity level

Reply:
Forgot to remove that. This was for debugging.